src/renderer/services/db/controller/impl/level.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/**
+ * @module db/controller/impl
+ */
+
+import { LevelUp } from 'levelup';
+import { IDatabaseController, ISearchOptions } from '../interface';
+import { EventEmitter } from 'events';
+// @ts-ignore
```

Comment:
you still need ignore even after you added level types?

Reply:
There are no types for `level` package, or at least I wasn't able to find them :/

Reply:
its ok then. I will create PR for fix of e2e tests with esModuleInterop flag. It will include a lot of style guide changes, hopefully it won't break things much